.github/workflows/docker-latest.yml

```diff
@@ -3,8 +3,7 @@ name: docker-latest
 on: [workflow_dispatch]
 
 env:
-  ORG: opendatacube
-  IMAGE: wps
+  IMAGE_NAME: opendatacube/wps
```

Comment:
i wonder if the name should be `datacube-wps` like how explorer and ows are named

Reply:
The Github action scan.yml file refers to 

`IMAGE_NAME: opendatacube/wps`

Reply:
I would agree with @pindge , we should probably change this to `datacube-wps` everywhere.

Reply:
but this PR can go on without it 👍🏽